Makefile

```diff
@@ -30,6 +30,7 @@ build_dep:
 	go get -u github.com/kisielk/errcheck
 	go get -u golang.org/x/tools/cmd/goimports
 	go get -u github.com/golang/lint/golint
+	go get -u github.com/davecheney/xattr
```

Comment:
You shouldn't need to do this - these extra imports are for the checkers/linters etc stuff that is included in an import should be automatically found.